### PR TITLE
Ageresnet

### PR DESCRIPTION
**Pull request by shuaijiang**
support age prediction

### CODE REVIEW

File: athena/main.py [hunk 1]
```diff
@@ -47,7 +47,9 @@
     "stargan" : StarganModel,
     "tts_transformer": TTSTransformer,
     "fastspeech": FastSpeech,
-    "speaker_resnet": SpeakerResnet
+    "speaker_resnet": SpeakerResnet,
+    "gender_resnet": SpeakerResnet,
```
Comment (Some-random) [inline athena/main.py:51]: We should probably change the name of SpeakerResnet to something like ClassificationResnet and use the keyword "task" to distinguish between each task

File: athena/solver.py [hunk 2]
```diff
@@ -335,6 +335,88 @@ def inference(self, dataset, rank_size=1):
         logging.info("model computation elapsed: %s\ttotal seconds: %s\tRTF: %.4f"
                      % (total_elapsed, total_seconds, float(total_elapsed / total_seconds)))
 
+class GenderSolver(BaseSolver):
```
Comment (Some-random) [inline athena/solver.py:338]: The only difference between GenderSolver and SpeakerClassificationSolver is the number of "top n" we choose. We should probably combine these two solvers into one "ClassificationSolver" and pass top_n as an argument.

Comment (neneluo) [reply athena/solver.py:338]: I think it's okay to merge now. I'll remove these duplicate codes before merging them to the master branch.

File: athena/metrics.py [hunk 3]
```diff
@@ -134,18 +134,18 @@ def __call__(self, logits, samples, logit_length=None):
 
 class ClassificationAccuracy(Accuracy):
     """ ClassificationAccuracy
-        Implements top-k accuracy calculation for speaker classification
+        Implements top-1 accuracy calculation for speaker classification
```
Comment (neneluo) [inline athena/metrics.py:137]: These codes should remain unchanged.

**Review by neneluo — APPROVED** [review]

**Pull request merged**